### PR TITLE
Update Force.wurst

### PR DESCRIPTION
**Pull request by GetLocalPlayer**

### CODE REVIEW

File: wurst/_handles/Force.wurst [hunk 1]
```diff
@@ -42,3 +45,55 @@ public function force.containsUnit(unit whichUnit) returns boolean
...
+
+init
+	for i=0 to bj_MAX_PLAYER_SLOTS-1
+		let p = Player(i)
+		allPlayers.addPlayer(p)
+		if GetPlayerController(p) == MAP_CONTROL_USER and GetPlayerSlotState(p) == PLAYER_SLOT_STATE_PLAYING
+			userPlayers.addPlayer(p)
+
+	let trg = CreateTrigger()
+	TriggerAddAction(trg, ()->userPlayers.removePlayer(GetTriggerPlayer()))
```
Comment (Frotty) [inline wurst/_handles/Force.wurst]: use extension func and use proper spacing

**Comment by Frotty** [comment]
should be unit tested. Also doesnt one usually just use ForForce?

**Comment by GetLocalPlayer** [comment]
> should be unit tested

But how?
screenshot_9

>  Also doesnt one usually just use ForForce?

Which cannot capture variables.

**Comment by Frotty** [comment]
> If further natives need to be implemented for compiletime, they can be requested via a Feature Request.

From (CONTRIBUTIN)[https://github.com/wurstscript/WurstStdlib2/blob/master/CONTRIBUTING.md#L12]

**Comment by GetLocalPlayer** [comment]
I see

File: wurst/_handles/Force.wurst [hunk 2]
```diff
@@ -42,3 +47,55 @@ public function force.containsUnit(unit whichUnit) returns boolean
...
+		return p
+
+	function close()
+		destroy this
+
+public function force.iterator() returns ForceIterator
+	return new ForceIterator(this)
+
+init
+	for i=0 to bj_MAX_PLAYER_SLOTS-1
```
Comment (Frotty) [inline wurst/_handles/Force.wurst]: leave spaces between literals and operators, ie. `i = 0`

Same below for `()->` should be `() ->`.

**Comment by Frotty** [comment]
Also, thinking about it, forces are clunky and lack a lot of funtionality - in my maps and Frentity i always expose a LinkedList<player> instead.

**Comment by GetLocalPlayer** [comment]
I've rethought some points and have come to the conclusion that `Force` package requires more than a little update. There are a lot of things to improve. If you don't mind I close this PR.

**Comment by Frotty** [comment]
Please elaborate on that so we can discuss it :)

**Pull request closed without merging**